### DISCUSSION TITLE
just a small question

### DISCUSSION THREAD

**Discussion by wenahyuck** [Cyfrin/foundry-full-course-cu#2616] [Q&A]
When I'm trying to deply a small contract on sepolia, it is giving me this message : -
Gas estimation errored with the following message (see below). The transaction execution will likely fail. Do you want to force sending?
Returned error: contract creation code storage out of gas
[object Object] 
Should I go ahead regardless?

**Comment by felixoder** [comment]
No that means that you do not have much sepolia for the gas fee. It often occurs when you have less money [test]. Go to sepolia faucet and grab some money and try to deploy again. That should work for sure. 
[make sure that you have enough money for the gas charges] 
there you go!

**Reply by ccvasco** [reply]
It happened the same to me. I selected force sending and the fee required was 1.43ETH. The code was the same as in the lesson. 
do test blockchains get clogged up as well?

**Reply by felixoder** [reply]
@ccvasco Yes sometimes they do so. The gas fees often gets higher that because of the usage of the blockchain at that specific time. What I personally do is I collect faucets everyday from every source and try to maintain approx 1 ETH but as u have said sometimes they got even higher so in that case I just leave that.

See the issue #1776

**Reply by wenahyuck** [reply]
Thanks guys, seems I had less sepolina eth that's why couldn't deploy.

**Comment by EngrPips** [comment]
Please share the contract you are trying to deploy.

**Reply by wenahyuck** [reply]
// SPDX-License-Identifier: MIT 

pragma solidity 0.8.18;  // solidity version 

contract SimpleStorage { 

    //favorite number gets initialized to 0 if no value is given 
    uint256  myFavNumber;

    struct Person {
        uint favNumber;
        string name;
    }
    
    // Dynamic array 
    Person[] public listOfPeople;

    //messi => 10 
    mapping (string => uint256) public nameToFavNumber;

    function store (uint256 _favNumber) public {

        myFavNumber = _favNumber;

    } 

    //view, pure and retieve functions
    function retieve() public view returns (uint256) {
        return myFavNumber;
    }
   
    //calldata, memory, storage
    function addPerson(string memory _name, uint256 _favNumber) public{

        listOfPeople.push(Person(_favNumber, _name));

        nameToFavNumber[_name] = _favNumber;
    }
    
}

**Reply by EngrPips** [reply]
I was able to deploy the code successfully. Maybe refresh your browser, then retry

<img width="1440" alt="Screenshot 2024-10-04 at 20 50 10" src="https://github.com/user-attachments/assets/80289f9a-9c0e-4322-8765-18050290281b">

**Reply by wenahyuck** [reply]
Did you deploy using your wallet?

**Reply by wenahyuck** [reply]
Thanks man, was able to deploy.

**Reply by EngrPips** [reply]
Interesting, Keep crushing it buddy.
